lintreview/config.py

```diff
@@ -1,9 +1,14 @@
+from __future__ import absolute_import
 import os
 import logging.config
 
 from flask.config import Config
-from ConfigParser import ConfigParser
-from StringIO import StringIO
+from six.moves.configparser import ConfigParser
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
Have you considered using [futurize](http://python-future.org/overview.html) it has import shims that make it easier to straddle 2 & 3.